win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1123,8 +1137,14 @@ BITMAP_LoadImageW(
         pbmi = LockResource(hgRsrc);
         if(!pbmi)
             return NULL;
+        ResSize = SizeofResource(hinst, hrsrc);
     }
 
+    if (pbmi->bmiHeader.biCompression == BI_BITFIELDS &&
+        pbmi->bmiHeader.biBitCount == 32 &&
+        pbmi->bmiHeader.biSizeImage + pbmi->bmiHeader.biSize + 12 != ResSize)
+        WARN("Possibly bad resource size provided\n");
```

Comment:
Does this occur?
If yes, does it cause out-of-bounds read operations?

Reply:
This occurs every time that these resource bitmaps are loaded when running with GCC compiled code. I added this as a check to confirm that we had this special condition for which GCC's resource compiler windres.exe fails. This is never triggered in an MSVC build. This is now redundant and duplicates the warning/error shown in Line 1184 from above, therefore I am now updating this PR to delete this complete "if" code block. Thanks.